README.md

```diff
@@ -78,7 +78,7 @@ There are two steps to detect duplicates:
 The dataset can be generated using the [OpenReview python API](https://github.com/openreview/openreview-py) which should be installed when this repository is installed. You can generate your own dataset from some other source as long as it is compliant with the format shown in the Datasets section.
 Start by creating an "experiment directory" (`experiment_dir`), and a JSON config file (e.g. `config.json`) in it. Go to the Configuration File section for details on how to create the `config.json`.
 
-Duplicate detection uses ELMo exclusively, since we normalize the scores for BM25. ELMo scores have values from 0 to 1. The closer a score is to 1, the more similar the submissions are.
+Duplicate detection uses ELMo exclusively, since we always normalize the scores for BM25. ELMo scores have values from 0 to 1. The closer a score is to 1, the more similar the submissions are. The `normalize` option for ELMo is disabled for duplicate detection.
```

Comment:
Thanks for the readme changes @carlosmondra . Can you also add an example with the `normalize` option. Right now I can see in the tests that this option is available but the readme does not mention how or where to use it.

Reply:
Sure!

Reply:
I have added an example and an explanation. Let me know what you think.